lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/RunningTaskBridge.java

```diff
@@ -47,6 +47,14 @@ public void send(final byte[] message) {
     jrunningTask.send(message);
   }
 
+  public void suspend(final byte[] message) {
+    if (message != null) {
+      jrunningTask.suspend(message);
+    } else {
+      jrunningTask.suspend();
```

Comment:
If this is the only place that suspend() is called, may be you can only keep public void Suspend(byte[] message) in RunningTask